reference-architecture/azure-ansible/onevm.sh

```
@@ -0,0 +1,269 @@
+export AUSERNAME=$2
+export PASSWORD=$3
+export HOSTNAME=$4
+export NODECOUNT=$5
+export ROUTEREXTIP=$6
+export RHNUSERNAME=$7
+export RHNPASSWORD=$8
+export RHNPOOLID=$9
+export SSHPRIVATEDATA=${10}
+export SSHPUBLICDATA=${11}
+export SSHPUBLICDATA2=${12}
+export SSHPUBLICDATA3=${13}
+
+ps -ef | grep onevm.sh > cmdline.out
+
+mkdir -p /home/$AUSERNAME/.ssh
+echo $SSHPUBLICDATA $SSHPUBLICDATA2 $SSHPUBLICDATA3 >  /home/$AUSERNAME/.ssh/id_rsa.pub
+echo $SSHPRIVATEDATA | base64 --d > /home/$AUSERNAME/.ssh/id_rsa
+chown $AUSERNAME /home/$AUSERNAME/.ssh/id_rsa.pub
+chmod 600 /home/$AUSERNAME/.ssh/id_rsa.pub
+chown $AUSERNAME /home/$AUSERNAME/.ssh/id_rsa
+chmod 600 /home/$AUSERNAME/.ssh/id_rsa
+
+mkdir -p /root/.ssh
+echo $SSHPRIVATEDATA | base64 --d > /root/.ssh/id_rsa
+echo $SSHPUBLICDATA $SSHPUBLICDATA2 $SSHPUBLICDATA3   >  /root/.ssh/id_rsa.pub
+chown root /root/.ssh/id_rsa.pub
+chmod 600 /root/.ssh/id_rsa.pub
+chown root /root/.ssh/id_rsa
+chmod 600 /root/.ssh/id_rsa
+
+sleep 30
+cat <<EOF > /root/setup_ssmtp.sh
+# \$1 = Gmail Account (Leave off @gmail.com ie user)
+# \$2 = Gmail Password
+# \$3 = Notification email address
+# Setup ssmtp mta agent for use with gmail
+yum -y install wget
+wget -c https://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-8.noarch.rpm
+rpm -ivh epel-release-7-8.noarch.rpm
+yum -y install ssmtp
+alternatives --set mta  /usr/sbin/sendmail.ssmtp
+systemctl stop postfix
+systemctl disable postfix
+cat <<EOFZ > /etc/ssmtp/ssmtp.conf
+root=postmaster
+mailhub=mail
+TLS_CA_File=/etc/pki/tls/certs/ca-bundle.crt
+mailhub=smtp.gmail.com:587   # SMTP server for Gmail
+Hostname=localhost
+UseTLS=YES
+UseSTARTTLS=Yes
+FromLineOverride=YES #TO CHANGE FROM EMAIL
+Root=\${3} # Redirect root email
+AuthUser=\${1}@gmail.com
+AuthPass=\${2}
+AuthMethod=LOGIN
+RewriteDomain=gmail.com
+EOFZ
+cat <<EOFZ > /etc/ssmtp/revaliases
+root:\${1}@gmail.com:smtp.gmail.com:587
+EOFZ
+EOF
+chmod +x /root/setup_ssmtp.sh
+# Ignore Error If It Dont work
```

"Ignore Error If It Dont work"
Maybe something like "Continue even if ssmtp.sh script errors out"